### DISCUSSION TITLE
Idk what im supposed to do when it says this:

### DISCUSSION THREAD

**Discussion by marindeveloper** [KadeArchive/Kade-Engine#529] [Q&A]
I get this whenever i compile the game:
C:\Users\user\Desktop\Marinluvcat's Animations and Videos\some stuff\Kade-Engine-0.2.0>lime test windows
source/PlayState.hx:99: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
Sys_error("export/release/windows/obj/Options.txt: Permission denied")

C:\Users\user\Desktop\Marinluvcat's Animations and Videos\some stuff\Kade-Engine-0.2.0>

My ver is 0.2.0 and im using this ver bc polymod has errors for me lik when i compile the game in earlier ver it doesnt work and it says that polymod in the code is not the right thing to use for some reason

if anyone knows how to fix this then please reply to this.

**Comment by marindeveloper** [comment]
Edit: The same thing happenes even in the newest ver

**Comment by shaderw0lf** [comment]
it's ok how i think
i'm building kade engine with the same warnings but nothing terrible happens
